core/os/android/device.go

```diff
@@ -90,6 +90,8 @@ type Device interface {
 	DeleteSystemSetting(ctx context.Context, namespace, key string) error
 	// StartPerfettoTrace starts a perfetto trace.
 	StartPerfettoTrace(ctx context.Context, config *perfetto_pb.TraceConfig, out string, stop task.Signal) error
+	// DeleteFile deletes a file on the device.
+	DeleteFile(ctx context.Context, traceFile string) error
```

Comment:
if I understand correctly, the `trace.go` change is not necessary. Hence no need to extend the interface